src/tools/cut_hole_tool.h

```diff
@@ -35,11 +35,14 @@ class QMouseEvent;
 class QPainter;
 class QRectF;
 
+namespace OpenOrienteering {
+
+class DrawLineAndAreaTool;
 class MapCoordF;
 class MapEditorController;
 class MapWidget;
 class PathObject;
-class DrawLineAndAreaTool;
+
```

Comment:
This is another instance of the phenomenon... Why has the empty line been added?